### PR TITLE
Add libnfnetlink/1.0.1

### PR DESCRIPTION
**Pull request by skannan89**
Specify library name and version:  **libnfnetlink/1.0.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libnfnetlink/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        autotools = self._configure_autotools()
+        autotools.install()
+
+        tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "etc"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
```
Comment (prince-chrismc) [inline recipes/libnfnetlink/all/conanfile.py:60]: You should explicit set the pkg_config name

**Review by prince-chrismc — APPROVED** [review]

File: recipes/libnfnetlink/all/conanfile.py (hunk 1 above)
Comment (skannan89) [reply recipes/libnfnetlink/all/conanfile.py:60]: Thanks for reviewing! Where do I set the pkg_config name? in `package_info`? 
Will something like this work: 
> self.cpp_info.names["pkg_config"] = "libnfnetlink"

Comment (prince-chrismc) [reply recipes/libnfnetlink/all/conanfile.py:60]: Yes

https://github.com/conan-io/conan-center-index/blob/2b4c8896953ad9c811a1bff4482bdfce8a0cf7bd/recipes/wayland/all/conanfile.py#L82

Comment (skannan89) [reply recipes/libnfnetlink/all/conanfile.py:60]: Thanks! Done.

**Review by intelligide — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**